src/odemis/driver/focustracker.py

```diff
@@ -0,0 +1,215 @@
+# -*- coding: utf-8 -*-
+"""
+Created on 2 Jul 2019
+
+@author: Thera Pals
+
+Copyright © 2012-2019 Thera Pals, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+import logging
+import os
+
+import canopen  # TODO package canopen and add to project requirements.
+
+import odemis
+from odemis import model
+
+
+class FocusTrackerCO(model.HwComponent):
+
+    def __init__(self, name, role, channel, node_idx, **kwargs):
+        """
+
+        channel (str): channel name of can bus
+        node_idx (int): node index of focus tracker
+        """
+        model.HwComponent.__init__(self, name, role, **kwargs)
+        # Connect to the CANbus and the CANopen network.
+        self.network = canopen.Network()
+        self.network.connect(bustype='socketcan', channel=channel)
+        self.network.check()
+        self.node = canopen.RemoteNode(node_idx, os.path.dirname(odemis.__file__) + "/driver/FocusTracker.eds")
+        self.network.add_node(self.node)
```

Comment:
In `swVersion`, you could indicate the python-canopen/python-can versions. 
If you have some way to get information on the USB can adapter, it would be nice. Maybe `self.network.bus.channel_info` reports something non-boring?

Reply:
`self.network.bus.channel_info`  reports `'socketcan channel \'can0\''` so nothing interesting, I also can't find anything interesting. For now I've indicated the python-canopen/python-can versions in `swVersion`.